src/main/java/com/google/sps/servlets/ManageListServlet.java

```diff
@@ -55,31 +59,58 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String type = request.getParameter("type").toLowerCase();
     String action = request.getParameter("action").toLowerCase();
 
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Are you using this `datastore` variable anywhere in this `doPost`? I think you can remove it.